include/swift/Option/FrontendOptions.td

```diff
@@ -306,6 +306,10 @@ def disable_reflection_names : Flag<["-"], "disable-reflection-names">,
   HelpText<"Disable emission of names of stored properties and enum cases in"
            "reflection metadata">;
 
+def function_sections: Flag<["-"], "function-sections">,
+  Flags<[FrontendOption, NoInteractiveOption, DoesNotAffectIncrementalBuild]>,
```

Comment:
This should impact the incremental builds I think.

Reply:
I think you're right, done.